mcgill-augmented-reality-backend/src/main/java/mcgillar/backend/controllers/user/AppUserController.java

```diff
@@ -67,4 +69,22 @@ public ResponseEntity<ValidatedUser> changeUsername(@RequestParam String newUser
         return new ResponseEntity<ValidatedUser>(user, HttpStatus.ACCEPTED);
     }
 
+    /**
+     * 
+     * @param authentication
+     * @return deleted user
+     * 
+     * !! Danger !!
+     */
+    @DeleteMapping("delete-account")
```

Comment:
This isnt exactly in the scope of the ticket but we can keep it hush

Reply:
You are right, my bad